lerna.json

```diff
@@ -5,7 +5,7 @@
   "packages": ["packages/*", "docs"],
   "command": {
     "version": {
-      "allowBranch": "master",
+      "allowBranch": ["master", "marco/prerelease"],
```

Comment:
> we won't be able to do pre-releases on other branches rather than master?

No, as it is right now.

> Or we will, but only if we change this value on that branch with its own name, right?

If you tweak this value you could do it, as it's described in the PR description.

But the idea behind this PR is to be able to release alpha versions from master, so eager teams can test them out. We can introduce breaking changes in new alphas and once we're ready release a stable version.